services/commands-service.ts

```diff
@@ -62,6 +73,11 @@ export class CommandsService implements ICommandsService {
 				} else {
 					command.execute(commandArguments).wait();
 				}
+				
+				if (this.commandHelp[commandName]) {
```

Comment:
`commandHelp` function returns the content of `this.$staticConfig.COMMAND_HELP_FILE_NAME` but here we use it as dictionary.. I'm confused :confused: